src/gui/src/components/analyze/ReportItem.vue

```diff
@@ -30,6 +30,15 @@
       >
         {{ $t('button.save') }}
       </v-btn>
+      <v-btn
+        prepend-icon="mdi-content-copy"
+        color="primary"
+        variant="flat"
+        class="ml-4"
+        @click="cloneReportItem"
+      >
+        Clone
+      </v-btn>
```

Comment:
It could be an improvement to have the button visible only when the report is already saved. Just like with the switch "Completed"

Reply:
good idea, changed it